packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -547,6 +553,68 @@ test.describe('PieInput - Component tests', () => {
                 });
             });
         });
+
+        test.describe('assistiveText', () => {
+            test('should not render the assistive text component if the prop is not provided', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieInput, {});
+
+                // Act
+                const assistiveText = page.locator(assistiveTextSelector);
+
+                // Assert
+                expect((assistiveText)).not.toBeVisible();
+            });
+
+            test('should render the assistive text component if the prop is provided', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieInput, {
+                    props: {
+                        assistiveText: 'Default text',
+                    } as InputProps,
+                });
+
+                // Act
+                const assistiveText = page.locator(assistiveTextSelector);
+
+                // Assert
+                expect(assistiveText).toBeVisible();
+            });
+
+            test('should render the assistive text component with the default variant if "status" is not provided', async ({ mount, page }) => {
```

Comment:
Not a massive issue so can leave as is but i'd be tempted to merge this test with the one above as it is mostly the same apart from the default check. You can render two expects in the single test and just update the description e.g. `should render the assistive text component if the prop is provided with the default variant if "status" is not provided`

Then the next test could be:

```javascript
describe('Assistive text: Status')
   statusTypes.forEach((status) => {
       test(`should render with the ${status} variant`, async ({ mount, page }) => {